project.clj

```diff
@@ -56,9 +56,9 @@
                  [slingshot "0.12.2"]
 
                  ;; Database connectivity
-                 [com.zaxxer/HikariCP "2.4.3"]
+                 [com.zaxxer/HikariCP "2.4.5"]
                  [org.clojure/java.jdbc "0.4.2"]
-                 [org.postgresql/postgresql "9.4-1206-jdbc4"]
+                 [org.postgresql/postgresql "9.4.1208.jre7"]
```

Comment:
These are versions bumps which align our dependencies with the versions used by RBAC, RBAC-client, and activity.